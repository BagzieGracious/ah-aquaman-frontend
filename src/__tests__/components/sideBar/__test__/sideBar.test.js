import React from "react";
import { shallow } from "enzyme";
import Sidebar from "../../../../components/article/sidebar";

describe("Sidebar component test", () => {
  const wrapper = shallow(<Sidebar />);

  it("Should render without crashing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
