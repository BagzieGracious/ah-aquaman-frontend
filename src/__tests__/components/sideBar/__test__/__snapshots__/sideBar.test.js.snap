// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar component test Should render without crashing 1`] = `ShallowWrapper {}`;
